### DISCUSSION TITLE
How do I try latest core from github?

### DISCUSSION THREAD

**Discussion by 877dev** [esp8266/Arduino#8436] [Q&A] [answered]
Sorry for the basic question, I want to try the latest code to see if it resolves a problem with a Wemos ESP8266 device, as per the "New Issues" guide. Currently using ESP8266 v3.0.2 from board manager within Arduino IDE.

Do I replace files in this folder (Windows 10, Arduino IDE latest):
`C:\Users\username\AppData\Local\Arduino15\packages\esp8266\hardware\esp8266\3.0.2`

**Comment by JAndrassy** [comment, marked as answer]
https://github.com/esp8266/Arduino#using-git-version

**Reply by 877dev** [reply]
> https://github.com/esp8266/Arduino#using-git-version

Thank you!
